bot/exts/evergreen/wikipedia.py

```diff
@@ -1,114 +1,99 @@
-import asyncio
-import datetime
 import logging
+import re
+from datetime import datetime
+from enum import Enum
+from html import unescape
 from typing import List, Optional
 
-from aiohttp import client_exceptions
-from discord import Color, Embed, Message
+from discord import Color, Embed
 from discord.ext import commands
 
-from bot.constants import Wikipedia
+from bot.bot import Bot
+from bot.utils import LinePaginator
 
 log = logging.getLogger(__name__)
 
-SEARCH_API = "https://en.wikipedia.org/w/api.php?action=query&list=search&srsearch={search_term}&format=json"
-WIKIPEDIA_URL = "https://en.wikipedia.org/wiki/{title}"
+SEARCH_API = "https://en.wikipedia.org/w/api.php?action=query&list=search&prop=info&inprop=url&utf8=& \
+      format=json&origin=*&srlimit={number_of_results}&srsearch={string}"
+WIKI_THUMBNAIL = "https://upload.wikimedia.org/wikipedia/en/thumb/8/80/Wikipedia-logo-v2.svg" \
+                 "/330px-Wikipedia-logo-v2.svg.png"
```

Comment:
I think using () to write multiline strings looks better.
```suggestion
SEARCH_API = (
    "https://en.wikipedia.org/w/api.php?action=query&list=search&prop=info&inprop=url&utf8=&"
    "format=json&origin=*&srlimit={number_of_results}&srsearch={string}"
)
WIKI_THUMBNAIL = (
    "https://upload.wikimedia.org/wikipedia/en/thumb/8/80/Wikipedia-logo-v2.svg"
    "/330px-Wikipedia-logo-v2.svg.png"
)
```